doc/api/source/index.html.md.erb

```diff
@@ -16,21 +16,54 @@ title: About
 
 ## What does this API do?
 
-This API provides access to metrics about the performance of GOV.UK content.
-Each metric is a time series, so you can see how the content performance has
-changed over time.
+This API provides access to time series metrics data about the performance of GOV.UK content. Examples of metrics include unique page views and number of PDF attachments. Refer to the [getting started section][] for a complete list of metrics.
 
-This API does not expose the content itself. For that you should use the [GOV.UK Content API][].
+This API does not expose the content itself. Use the [GOV.UK Content API][] to expose the content.
```

Comment:
I think this should read "to access the content" instead of "to expose the content"?

Reply:
done.